### ISSUE TITLE
Create "Get 3D Model" Pop-up on 3d.io instead of redirect to Spaces

### ISSUE THREAD

**Issue by dburgauer** [archilogic-com/3dio-website#45]
Currently clicking on the "Get 3D Models" CTA redirects to https://spaces.archilogic.com/explore?popup=order

This might be confusing to the first-time visitor (Archilogic vs. 3d.io). It also creates a disconnect in the user experience as they are now on Spaces and not 3d.io. Lastly, this skews our sign-up stats for 3d.io as these sign-ups are recorded on Spaces but originate from 3d.io.

Suggest to have a pop-up window on 3d.io for model orders.

**Comment by tomas-polach** [comment]
good point! i suggest to be consistent with our product structure (products = APIs):
- every API has a miniapp and awesome documentation

for floor plan conversion it should be:
- APP: convert-floor-plan.3d.io

the mini app should also have links to 3d.io, github, stack overflow - like https://furniture.3d.io has

the current (temporary) URL is: https://io3d-floor-plan-app.herokuapp.com
